options.c

```diff
@@ -80,6 +97,20 @@ int init_options(int opt_cnt, char *opt_arr[], struct tool_options *options)
 	return rc;
 }
 
+static int verify_target(struct tool_options *options, int target)
+{
+	if (options->target != INVALID) {
+		print_error("duplicated operate target.");
+		goto out;
+	}
+
+	options->target = target;
```

Comment:
target should be either DME_PEER or DME_LOCAL, better set those as enum and check it here